lib/api/apiUtils/object/tagging.js

```diff
@@ -161,3 +167,46 @@ export function convertToXml(objectTags) {
     xml.push('</TagSet> </Tagging>');
     return xml.join('');
 }
+
+export function parseTagFromQuery(tagQuery) {
+    const tagsResult = {};
+    const pairs = tagQuery.split('&');
+    let key;
+    let value;
+    let emptyTag = 0;
+    if (pairs.length === 0) {
+        return tagsResult;
+    }
+    for (let i = 0; i < pairs.length; i++) {
+        const pair = pairs[i];
+        if (!pair) {
+            emptyTag ++;
```

Comment:
This test with empty tags passed against AWS: https://github.com/scality/S3/pull/727/files#diff-1bf0b1ddd89f1b1393852bd98268596fR167